.github/workflows/build.yml

```diff
@@ -270,7 +270,7 @@ jobs:
           set -eo pipefail
           yarn build:macos || yarn build:macos
           pod install --project-directory=macos
-          ../scripts/xcodebuild.sh macos/Example.xcworkspace build-for-testing
```

Comment:
Not a permanent change, but removing this action allowed this command to pass locally for me. I wanted to confirm that is true if I passed it to CI.

Reply:
Would it make sense to add another step to build arm64?

Reply:
Ended up going with this suggestion. Added a step to the macOS job to also build arm64.